### PR TITLE
Fix alignment of `show-names` in review comments

### PR DESCRIPTION
**Pull request by cheap-glitch**
## Test URLs
Any PR with review comments, e.g. https://github.com/sindresorhus/refined-github/pull/4023

## Screenshots

<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td>
		
	<td>
		
</table>

### CODE REVIEW

**Review by fregante — CHANGES_REQUESTED** [review]
> Timeline event:

This is worse than before. I can't quite tell in the other screenshots since they're cut too close

File: source/features/show-names.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ async function init(): Promise<false | void> {
 				insertionPoint.after(
 					' (',
 					<bdo className="css-truncate">
-						<span className="css-truncate-target" style={{maxWidth: '200px'}}>
+						<span className="css-truncate-target v-align-middle" style={{maxWidth: '200px'}}>
```
Comment (fregante) [inline source/features/show-names.tsx:59]: The right alignment for text is `baseline`

Comment (cheap-glitch) [reply source/features/show-names.tsx:59]: That's what I thought too, but it's even worse. I tried every `vertical-align` value and `middle` is the ~best~ least worst compromise. We lose a few pixels of alignment in the comment headers and timeline events, but we get much better alignment in review headers.

We could also try to add some padding or a `transform: translate()`

Comment (fregante) [reply source/features/show-names.tsx]: No, baseline is the correct one and it looks fine where I tried it. What's pulling the element out of the alignment is the avatar. Transform would just be tape to hold it together temporarily

Comment (fregante) [reply source/features/show-names.tsx]: Baseline on review comment

Screen Shot

Comment (fregante) [reply source/features/show-names.tsx]: Baseline on review request (zero change, it's the default)

Screen Shot

Comment (fregante) [reply source/features/show-names.tsx]: Baseline on regular comments (zero change, it's the default)

Screen Shot

Comment (yakov116) [reply source/features/show-names.tsx]: I see the issue on review comments

image

Comment (fregante) [reply source/features/show-names.tsx]: My screenshots are with `baseline` applied. I also see the issue on `main`

Comment (cheap-glitch) [reply source/features/show-names.tsx]: > No, baseline is the correct one and it looks fine where I tried it.

On what element did you apply `vertical-align`? I can't get it to align properly :thinking:

Comment (fregante) [reply source/features/show-names.tsx]: On this same element. However it works correctly on Safari, but in Chrome it doesn't, `overflow:hidden` throws it off. Investigating

Comment (fregante) [reply source/features/show-names.tsx]: `display: inline-block` on `bdo` fixes it everywhere in every browser

File: source/features/show-names.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ async function init(): Promise<false | void> {
 				const insertionPoint = usernameElement.parentElement!.tagName === 'STRONG' ? usernameElement.parentElement! : usernameElement;
 				insertionPoint.after(
 					' (',
-					<bdo className="css-truncate">
+					<bdo className="css-truncate" style={{display: 'inline-block'}}>
```
Comment (fregante) [inline source/features/show-names.tsx:58]:
```suggestion
					<bdo className="css-truncate d-inline-block">
```

**Pull request merged**